SpaceTracker/SpaceExtractor.cs

```diff
@@ -1,141 +1,170 @@
-﻿using System;
```

Comment:
seems like a lot of changes git has identified. However, the code looks pretty much similar.